src/utils/createMusicMetadata.ts

```diff
@@ -1,13 +1,15 @@
+import { FormData } from "@/components/MintForm"
+
 const music_metadata = {
-	name: '', // music nft
+	name: '', // artist name - title
 	description: '', // music NFT with data on-chain as Base64 encoded string.
-	image: '', // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
+	image: undefined, // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
 	version: '0.1',
-	title: '', // music nft
-	artist: '', // sweetman.eth
-	duration: 0, // 420
+	title: '', // title of song
+	artist: '', // artist name
+	duration: 0, // length of song as a float
```

Comment:
MUST be integer. floats are not accepted.

https://gist.github.com/bretth18/df8358c840fa94946ec212f753e290dd